.github/workflows/canary.yml

```diff
@@ -0,0 +1,152 @@
+name: Canary
+
+on:
+  push:
```

Comment:
When it all works, then move this to have it be triggered the same way that the compatibility tests are: From test workflow and only after linting and tests has passed

Reply:
Done ✔️